modules/compliance-apply-remediation-for-customized-mcp.adoc

```diff
@@ -1,9 +1,14 @@
 :_content-type: PROCEDURE
-[id="complianc-operator-apply-remediation-for-customized-mcp"]
+[id="compliance-operator-apply-remediation-for-customized-mcp"]
 = Applying remediation when using customized machine config pools
 
 When you create a custom `MachineConfigPool`, add a label to the `MachineConfigPool` so that `machineConfigPoolSelector` present in the `KubeletConfig` can match the label with `MachineConfigPool`.
 
+[IMPORTANT]
+====
+Do not set `protectKernelDefaults: false` in the `KubeletConfig` file, as the `MachineConfigPool` object might fail to unpause and hang unexpectedly after the Compliance Operator finishes applying remediation.
```

Comment:
* as -> because (per ISG)
* It might be fine as is, but I'm almost wondering if you could read it as the MCP could fail to unpause and fail to hang unexpectedly. Just wondering if it's important to say both of those, or if just saying one of them would be sufficient (that it fails to unpause or that it hangs unexpectedtly)? Again, fine to keep it as is, just something to think about!